### PR TITLE
docs: convert code images to code fences

### PR DESCRIPTION
**Pull request by ChristianMurphy**

### CODE REVIEW

File: guides/InstallationAdminGuide.md [hunk 1]
```diff
@@ -523,14 +531,32 @@ Where:
 
 An example directive is:
 
-![alt text](ExampleDirective.png "Example Directive")
+```apache
+<VirtualHost *:80>
+  ServerName www.equella.com
+  ProxyPass / http://equellahost:80/ nocanon
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: Might as well change these to the generic form as well

```apache
ServerName {external-server-name}
ProxyPass / http://{equella-host}:{http-port}/ nocanon
```

File: guides/InstallationAdminGuide.md [hunk 2]
```diff
@@ -539,7 +565,20 @@ Where:
 * ‘nocanon’ ensures URLs are passed through to the host without processing.
 
 An example directive is:
-![alt text](ExampleDirectiveSSL.png "Example Directive SSL")
+
+```apache
+<VirtualHost *:443>
+  ServerName www.equella.com
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: Might as well change these to the generic form as well

```
ServerName {external-server-name}
ProxyPass / http://{equella-host}:{http-port}/ nocanon
```

**Review by cbeach47 — CHANGES_REQUESTED** [review]
Thanks for cleaning this up :+1:

File: guides/InstallationAdminGuide.md (hunk 2 above)
Comment (ChristianMurphy) [reply guides/InstallationAdminGuide.md]: So duplicate the example on line 547?

File: guides/InstallationAdminGuide.md (hunk 1 above)
Comment (ChristianMurphy) [reply guides/InstallationAdminGuide.md]: Duplicate the example on line 518?

Comment (ChristianMurphy) [reply guides/InstallationAdminGuide.md]: Updated :ballot_box_with_check:

File: guides/InstallationAdminGuide.md (hunk 2 above)
Comment (ChristianMurphy) [reply guides/InstallationAdminGuide.md]: Updated :ballot_box_with_check:

Comment (cbeach47) [reply guides/InstallationAdminGuide.md]: yeah

**Review by cbeach47 — APPROVED** [review]

**Pull request merged**